modern/tests/fixtures/expected.py

```diff
@@ -114,6 +116,29 @@ def get_envfile_values():
     return env_values
 
 
+def get_system_libs_c():
+    return [
+        'libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0',
+        'libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6',
+        '/lib64/ld-linux-x86-64.so.2',
+    ]
+
+
+def get_system_libs_cpp(compiler):
+    if compiler == "gcc":
+        return [
+                'libstdc++.so.6 => /usr/local/lib64/libstdc++.so.6',
+                'libgcc_s.so.1 => /usr/local/lib64/libgcc_s.so.1',
+        ] + get_system_libs_c()
+    else:
+        return [
+            'libc++abi.so.1 => /usr/local/lib/libc++abi.so.1',
+            'libc++.so.1 => /usr/local/lib/libc++.so.1',
+            'libstdc++.so.6 => /usr/local/lib64/libstdc++.so.6',
+            'libllvm-unwind.so.1 => /usr/local/lib/libllvm-unwind.so.1',
+        ] + get_system_libs_c()
```

Comment:
I would try to avoid hardcoded values here. I know it is a `fixture` and sometimes they contain hardcoded values (data to populate testing databases,...) but in this case, everything here is parametrized. 

I would really move these lines to the only test where they are used. wdyt?

Reply:
fair enough

Reply:
Done